src/ui/InsertWordUI.js

```diff
@@ -78,22 +79,24 @@ export default class InsertWordUI {
 
   checkInputs() {
     if (!this.item.value) {
-      return null;
+      return false;
     }
 
     if (!this.itemDescription.value && !this.exampleSentence.value && !this.player.src) {
-      return null;
+      return false;
     }
+    return true;
   }
 
   createWord() {
+    const currUserId = JSON.parse(localStorage.getItem(LOCALSTORAGE_KEY));
```

Comment:
the `userId` should be passed to `InsertWordUI` from `insertWord.js`, which is the bridge among different resources.
Strictly speaking, how we obtain the `userId` should not be part of the `InsertWordUI` responsibility